src/pkg/task/sweep_manager.go

```diff
@@ -182,6 +185,32 @@ func (sm *sweepManager) Clean(ctx context.Context, execIDs []int64) error {
 	return nil
 }
 
+// FixDanglingStateExecution update executions always running
+func (sm *sweepManager) FixDanglingStateExecution(ctx context.Context) error {
```

Comment:
I didn't see where this method was called.

Reply:
Updated